internal/access/resource_access_workflow.go

```diff
@@ -190,8 +215,23 @@ func (r *AccessWorkflowResource) Create(ctx context.Context, req resource.Create
 
 	if data.Validation != nil {
 
-		createReq.Validation = &configv1alpha1.ValidationConfig{HasReason: data.Validation.HasReason.ValueBool()}
+		var regexValidations []*accessv1alpha1.RegexValidation
+
+		if data.Validation.Regex != nil {
+			for _, r := range data.Validation.Regex {
+				regexValidations = append(regexValidations, &accessv1alpha1.RegexValidation{
+					RegexPattern: r.RegexPattern.ValueString(),
+					ErrorMessage: r.ErrorMessage.ValueString(),
+				})
+			}
+		}
+
+		createReq.Validation = &configv1alpha1.ValidationConfig{
+			HasReason: data.Validation.HasReason.ValueBool(),
```

Comment:
There might be a possible nil pointer exception here if HasReason is not set in the validation?

Reply:
There wasn't a check for this previously because it was the only validation available I think